src/Domain/LeanCode.DomainModels.MassTransitRelay/Outbox/NewtonsoftJsonEventsSerializer.cs

```diff
@@ -47,23 +47,23 @@ private Type GetEventType(string type)
                 .FirstOrDefault()
                 ?? throw new InvalidOperationException($"Type {type} is not defined in any of the provided assemblies.");
         }
+    }
 
-        private class ContractResolver : DefaultContractResolver
-        {
-            protected override JsonProperty CreateProperty(MemberInfo member, MemberSerialization memberSerialization)
-                => MakeWriteable(base.CreateProperty(member, memberSerialization), member);
+    public class ContractResolver : DefaultContractResolver
```

Comment:
If we're making it public, let's rename it to something more descriptive, e.g. `NewtonsoftJsonContractResolver`.